src/iterator.vala

```diff
@@ -19,6 +19,12 @@ public class Vast.Iterator : Object
         get { return _offset; }
     }
 
+    public uint8 * origin {get; private set;}
```

Comment:
I would not use a property here because it's just forwarding the array's origin. It's okay to cache the origin pointer for faster access though.